### PR TITLE
[ISSUE-48][FEATURE][FOLLOW UP] Add controller component

### PR DESCRIPTION
**Pull request by wangao1236**
### What changes were proposed in this pull request?
For issue #48 
I add some codes about controller module, and I'll add more unit tests in next PR.

### CODE REVIEW

**Comment by jerqi** [comment]
@czy006 @zuston @thousandhu Could you help me review this patch?

File: deploy/kubernetes/operator/pkg/controller/constants/constants.go [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+	ShuffleServerServiceName = "server"
+
+	// ExcludeNodesFile indicates volume mounting name of exclude nodes file
+	ExcludeNodesFile = "exclude-nodes-file"
+
+	// UpdateStatusError means reason of updating status of rss error
+	UpdateStatusError = "UpdateStatusError"
+
+	// OwnerLabel is the label of configMap's owner.
+	OwnerLabel = "data.tencent.io/owner-label"
```
Comment (jerqi) [inline deploy/kubernetes/operator/pkg/controller/constants/constants.go:61]: Should we change this from `data.tencent.io` to `org.apache.uniffle`?

Comment (wangao1236) [reply deploy/kubernetes/operator/pkg/controller/constants/constants.go]: Sorry, I miss it, I have modified.

**Comment by jerqi** [comment]
Like discussion at dev mail list, we will freeze the code and cut 0.6 version branch in September 15, we  will not merge this pr before I cut 0.6 version branch, are you ok?

File: deploy/kubernetes/operator/pkg/controller/controller/rss.go [hunk 2]
```diff
@@ -0,0 +1,925 @@
...
+
+// processNextWorkRss deals with one rss key off the rssQueue, it returns false when it's time to quit.
+func (r *rssController) processNextWorkRss() bool {
+	rssKey, quit := r.rssQueue.Get()
+	if quit {
+		return false
+	}
+	defer r.rssQueue.Done(rssKey)
+
+	pvcNamespace, pvcName, err := cache.SplitMetaNamespaceKey(rssKey.(string))
```
[the remark this answers is not in the document]
Comment (wangao1236) [reply deploy/kubernetes/operator/pkg/controller/controller/rss.go]: Ok, I have modified this place.

File: deploy/kubernetes/operator/pkg/controller/controller/rss.go [hunk 3]
```diff
@@ -0,0 +1,925 @@
...
+		if apierrors.IsNotFound(err) {
+			klog.V(5).Infof("ignored deleted rss (%v/%v)", namespace, name)
+			return false, nil
+		}
+		klog.Errorf("get rss (%v/%v) failed: %v", namespace, name, err)
+		return false, err
+	}
+
+	if isDeletionCandidate(rss) {
+		return r.processDeleting(rss)
```
[the remark this answers is not in the document]
Comment (wangao1236) [reply deploy/kubernetes/operator/pkg/controller/controller/rss.go:348]: Ok, I have modified this place.

**Review by jerqi — APPROVED** [review]
LGTM, thanks @wangao1236 @thousandhu

**Pull request merged**